src/models/survey/surveyModels.ts

```diff
@@ -0,0 +1,22 @@
+export interface CommonProps {
+  currentStep: number;
+  totalSteps: number;
+  question: { id: number; text: string };
+  stepNumber: number;
+  teamMembers: string[];
+  options?: string[];
```

Comment:
어떤 옵션인지 주석이 있으면 좋을 것 같아요!
```suggestion
  options?: string[]; // ~~ 에 사용
```

Reply:
변경될 것 같아서 api 연동하면서 주석 같이 달아두겠습니다!